### ISSUE TITLE
Ways To Give - In Honor

### ISSUE THREAD

**Issue by etrepum** [MissionBit/missionbit.org#15]
[Fundraise Page Language](https://docs.google.com/document/d/1u8DQEzqyiEisfDJYvH0WM1Tc1vCjogP7pv9Pdx3Qbng/edit)

> Honor someone special by making a donation in their name.  Simply select “Dedicate my donation in honor or memory of someone” on the online donation form.  Or, indicate a tribute via mail to:
> Mission Bit
> 101A Clay St., #121 
> San Francisco, CA 94111

Blocked on: adding this field to the online donation form

**Comment by etrepum** [comment]
This is superseded by the new design
